docs/api-bzz.md

```diff
@@ -764,3 +755,12 @@ Calls `uploadFileFrom()` or `uploadDirectoryFrom()` depending on the provided `p
 1.  [`options?: UploadOptions = {}`](#uploadoptions)
 
 **Returns** `Promise<string>`
+
+### .uploadFileStream()
```

Comment:
This method should be documented above the [Node-specific APIs](https://github.com/MainframeHQ/erebos/pull/116/files#diff-e345420688e1aa1a0ed78fb28410a63bR664) now that it's also available for browser.